foladipo/docs-cabinet-cp2
# Chore/148098879/test frontend code

#### What does this PR do?

This PR adds tests for all the reducers and actions used in the frontend code. Tests for components will be added in a later chore.

#### Description of task to be completed

Add frontend tests

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Open a terminal/command prompt and `cd` into the directory of this app.

- Enter `npm run test:client` to run the frontend tests.

- Enter `npm run test:nyc:client` to see an analysis of the coverage of the frontend tests.

- Alternatively, you can visit [this link](https://coveralls.io/github/andela-foladipo/docs-cabinet-cp2) to see test coverage reports for this PR (and other branches too).


#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148098879

#### Screenshots

<img width="1134" alt="screen shot 2017-07-11 at 8 02 15 pm" src="https://user-images.githubusercontent.com/26963575/28085555-f9e07c04-6673-11e7-8427-3f2f7b9d4d0e.png">
